Overlord.Api/Program.cs

```diff
@@ -20,7 +20,12 @@ public static void Main(string[] args)
 
             builder.Services.AddMqttBrokerServices();
 
-            var connectionString = builder.Configuration.GetConnectionString("OverlordContext");
+            string? connectionString = builder.Configuration.GetConnectionString("OverlordContext");
```

Comment:
From a compiler perspective this generates the same code.
It is also redundant and doesn't add to the readability since it is very obvious what the type is.